src/flex/FlexView.js

```diff
@@ -68,6 +68,18 @@ const PropTypes = {
 @props(PropTypes, { strict: false })
 export default class FlexView extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
+  logWarnings = () => {
+    warn(() => {
+      if (this.props.basis === 'auto') {
+        return 'Default basis is auto. Force it to auto will leave shrink true as default';
```

Comment:
I would change first part of the sentence to
basis is "auto" by default: forcing it to "auto" ...
(I think we are already using double quotes for props in other components)